www/locales/en.json

```diff
@@ -76,9 +76,9 @@
       "ARJUN_DEV_JI": "Guru Arjun Dev Ji",
       "GOBIND_SINGH_SAHIB_JI": "Guru Gobind Singh Ji",
       "GRANTH_SAHIB_JI": "Guru Granth Sahib Ji",
-      "HAR_GOBIND_SAHIB_JI": "Guru Har Gobind Ji",
-      "HAR_KRISHAN_SAHIB_JI": "Guru Har Krishan Ji",
-      "HAR_RAI_SAHIB_JI": "Guru Har Rai Ji",
+      "HARGOBIND_SAHIB_JI": "Guru Hargobind Ji",
+      "HARKRISHAN_SAHIB_JI": "Guru Harkrishan Ji",
+      "HARRAI_SAHIB_JI": "Guru Harrai Ji",
```

Comment:
would there be push back? Is there other places where it's spelled this way?

Reply:
i think we should be fine here

Reply:
I've never seen this spelling before? Is this how it is spelled across other Khalis apps?

Reply:
i based it on the Gurmukhi from here:

ਪ੍ਰਿਥਮ ਭਗੌਤੀ ਸਿਮਰਿ ਕੈ ਗੁਰ ਨਾਨਕ ਲਈਂ ਧਿਆਇ ॥
ਫਿਰ ਅੰਗਦ ਗੁਰ ਤੇ ਅਮਰਦਾਸੁ ਰਾਮਦਾਸੈ ਹੋਈਂ ਸਹਾਇ ॥
ਅਰਜਨ ਹਰਿਗੋਬਿੰਦ ਨੋ ਸਿਮਰੌ ਸ੍ਰੀ ਹਰਿਰਾਇ ॥
ਸ੍ਰੀ ਹਰਿਕ੍ਰਿਸ਼ਨ ਧਿਆਈਐ ਜਿਸੁ ਡਿਠੇ ਸਭਿ ਦੁਖਿ ਜਾਇ ॥
ਤੇਗ ਬਹਾਦਰ ਸਿਮਰਿਐ ਘਰਿ ਨਉ ਨਿਧਿ ਆਵੈ ਧਾਇ ॥
ਸਭ ਥਾਈਂ ਹੋਇ ਸਹਾਇ ॥੧॥